docs/modules/operation/pages/deep-dive/admin/mib.adoc

```diff
@@ -6,33 +6,36 @@ They describe Simple Network Management Protocol (SNMP) elements for specific de
 By default, {page-component-title} includes an extensive list of MIB definitions from a number of vendors.
 You will only need to import MIB files for unformatted events as part of configuring SNMP traps and collection.
 
-You can parse SNMP MIB definitions and create {page-component-title} event definitions from any `TRAP-TYPE` and/or `NOTIFICATION-TYPE` macros they contain through the UI.
+You can parse SNMP MIB definitions and create {page-component-title} event definitions from any `TRAP-TYPE` or `NOTIFICATION-TYPE` macros that they contain through the UI.
 You can also generate data collection definitions from `OBJECT-TYPE` symbols defined in a MIB file.
 
-You can also use the `mib2opennms` command line utility to help create event definitions from MIB definitions.
+You can use the `mib2opennms` command line utility to help create event definitions from MIB definitions.
 Using the UI editor achieves similar results to using `mib2events`.
-We do not recommend using using `mib2opennms`, as it produces more bus errors than event definitions.
+
+NOTE: We do not recommend using using `mib2opennms`, as it produces more bus errors than event definitions.
 
 [[mib-import]]
+== Import MIB and create event definitions
 
 To import a MIB file and create event definitions, follow these steps:
 
 . Save the vendor MIB file to a location available to your local workstation.
 The MIB file does not need to be accessible by the server.
-. In {page-component-title}, click the gears icon and select *Additional Tools -> SNMP MIB Compiler*.
+. In {page-component-title}, click the *gears* icon at the top-right of the page.
+. Under Additional Tools, select *SNMP MIB Compiler*.
 . Click *Upload MIB*.
 . Select the MIB file and click *Open*.
 . Right-click the file and choose *Compile MIB*.
-. If the log area displays an error (usually a missing dependency), you need to correct the error to compile the MIB file successfully.
-.. Determine the type of error (dependency, file syntax).
-.. Google the name of the missing content, save it as a text file, upload it, and compile again.
-+
-AND/OR
+. If the log area displays an error (usually a missing dependency), you must correct the error to compile the MIB file successfully.
+** Determine the type of error (dependency, file syntax).
+** Google the name of the missing content, save it as a text file, upload it, and compile again.
```

Comment:
We should probably change this to "Search the Internet for" instead of "Google..."

Reply:
```suggestion
** Search the internet for the name of the missing content, save it as a text file, upload it, and compile again.
```